vllm/multimodal/inputs.py

```diff
@@ -594,11 +647,55 @@ class MultiModalKwargsItem(UserDict[str, MultiModalFieldElem]):
     def from_elems(elems: Sequence[MultiModalFieldElem]):
         return MultiModalKwargsItem({elem.key: elem for elem in elems})
 
-    @property
-    def modality(self) -> str:
+    def __init__(
+        self,
+        data: Optional[Mapping[str, MultiModalFieldElem]] = None,
+    ) -> None:
+        super().__init__(data)
+
         modalities = {elem.modality for elem in self.data.values()}
         assert len(modalities) == 1, f"Found different modalities={modalities}"
-        return next(iter(modalities))
+        self._modality = next(iter(modalities))
+
+        self._is_empty = False
+
+    @property
+    def modality(self) -> str:
+        return self._modality
+
+    @property
+    def is_empty(self) -> bool:
+        return self._is_empty
```

Comment:
```suggestion
    @property
    def is_empty(self) -> bool:
        return all(elem.data is None for elem in self.values())
```
I think whether mm_kwargs_item is empty should be inferred from its contents.

Reply:
Got it, let's leave it unchanged now!